root/statistics/Relationships.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {compare} from '../static/scripts/common/i18n';
+import {l_relationships} from '../static/scripts/common/i18n/relationships';
+import {l_statistics as l} from '../static/scripts/common/i18n/statistics';
+import {withCatalystContext} from '../context';
+import manifest from '../static/manifest';
+import formatEntityTypeName from '../static/scripts/common/utility/formatEntityTypeName';
+
+import {formatCount, formatPercentage} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+
+export type RelationshipsStatsT = {|
+  +$c: CatalystContextT,
+  +dateCollected: string,
+  +stats: {[string]: number},
+  +types: {[string]: RelationshipTypeT},
+|};
+
+declare type RelationshipTypeT = {|
+  +entity_types: $ReadOnlyArray<string>,
+  +tree: {[string]: Array<LinkTypeT>},
```

Comment:
AFAICT this needs to be Array, not $ReadOnlyArray, for it to be sortable, even if eslint would prefer it the other way.